### DISCUSSION TITLE
Update Options

### DISCUSSION THREAD

**Discussion by euf0ria** [Rem0o/FanControl.Releases#2396] [Ideas]
I always install the latest updates and would love if this could be automated.
Would it be possible to add an entry in the settings for handling updates?
Sub settings could be:
- Install updates automatically
- Restart application after update automatically
- Disable prompt when new update is available
- Check for updates every: x (day/week/month)
- Check for update button

**Comment by Rem0o** [comment]
* Check for update button ---> that exists already
* Disable prompt when new update is available -> that exists already

As for the automatic (silent) updating:

I fear putting that option in.   Updates *may* break functionality, which if the user is not conscious that an update just happened, may go unnoticed, and I can sense the horde of angry emails coming my way.

**Reply by Spuner** [reply]
> I fear putting that option in. Updates may break functionality, which if the user is not conscious that an update just happened, may go unnoticed, and I can sense the horde of angry emails coming my way.

After the update, a pop-up window may appear notifying you that Fan Control has been updated...

**Reply by Marcosaurios** [reply]
I was thinking about the option for "automated updates" today as well.

If I have it enabled I _don't mind_ breaking some functionality for a small % of the updates done. As far as the breaking changes are clear in the update changelog/message, I can fix that later.
Beautiful project, works like a charm and you can see a lot of effort put in every detail! Love it
